timelapse.py

```diff
@@ -62,8 +63,19 @@ def layer_changed(timelapse_folder, webcam_url, webcam_http_auth, webcam_https_v
     else:
         log_print('Failed to get timelapse snapshot.', file=sys.stderr)
 
+def create_video(timelapse_path, current_log_print, snapshots_path, keep_snapshots):
+    video_file = os.path.abspath(os.path.join(timelapse_path, current_log_print + ".mp4"))
+    snapshots_files = snapshots_path + os.path.sep + "*.jpg"
+    cmd = "ffmpeg -r 20 -y -pattern_type glob -i '" + snapshots_files + "' -vcodec libx264 " + video_file
+    os.system(cmd)
```

Comment:
Please use the modern `subprocess.run` instead.
https://docs.python.org/3/library/subprocess.html#subprocess.run